Vostok.Metrics.System/Host/DiskUsageInfo.cs

```diff
@@ -0,0 +1,16 @@
+﻿using JetBrains.Annotations;
+
+namespace Vostok.Metrics.System.Host
+{
+    [PublicAPI]
+    public class DiskUsageInfo
+    {
+        public string DiskName { get; set; }
+        public double IdleTimePercent { get; set; }
+        public double ReadLatency { get; set; }
```

Comment:
Latency measurement unit isn't obvious from this model.

Reply:
Should also probably provide a hint to what kind of aggregate this is (for library users). Average? Median? Some kind of quantile?